lib/tuya.js

```diff
@@ -1258,6 +1258,37 @@ const valueConverter = {
             return {voltage: (buffer[14] | buffer[13] << 8) / 10, current: (buffer[12] | buffer[11] << 8) / 1000};
         },
     },
+    phaseB: {
+        from: (v) => {
+            const buf = Buffer.from(v, 'base64');
+            return {
+                voltage: (buf[1] | buf[0] << 8) / 10,
+                current: (buf[4] | buf[3] << 8) / 1000,
+                power: (buf[7] | buf[6] << 8)};
+        },
+    },
+    alarm: {
+        from: (v) => {
+            const buffer = Buffer.from(v, 'base64');
+            const lookup = {
+                0: 'not set',
+                1: 'Over current threshold',
+                3: 'Over voltage threshold',
```

Comment:
Please make this values lower & snake case, e.g. `over_voltage_threshold`

Reply:
did it